src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+
+2.3.11
+++++++
+* Remove enable_cloud_console_aks_browse to enable 'az aks browse' by default
```

Comment:
Though this only really affects cloudshell, it is still a breaking change. Can you label it as such?